runner/src/server/plugins/engine/models/AdapterFormModel.ts

```diff
@@ -236,6 +236,7 @@ export class AdapterFormModel {
     }
 
     getRelevantPages(state: FormSubmissionState) {
+        console.log(`[AdapterFormModel] get relevant pages`);
```

Comment:
Can we remove this or change to a proper log message please

Reply:
Yes I added these since there were some issues & could not identify this will be removed